runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -1660,4 +1666,9 @@
 	 * Returns the max item index + 1 from cache
 	 */
 	int getItemCount();
+
+	/**
+	 * Makes all widgets behave as if they are {@link WidgetConfig#WIDGET_USE_TARGET}
+	 */
+	void setAllWidgetsAreOpTargetable(boolean value);
```

Comment:
Would it be better to just give it a mask to apply?

Reply:
I don't think there are any other bits are useful to set globally